README.md

```diff
@@ -413,7 +413,32 @@ const client = createClient({
 </details>
 
 <details id="apollo-client">
-<summary><a href="#apollo-client">🔗</a> Client usage with <a href="https://www.apollographql.com">Apollo</a></summary>
+<summary><a href="#apollo-client">🔗</a> Client usage with <a href="https://www.apollographql.com/docs/react/">Apollo Client Web</a> v3.5.10+</summary>
+
+```typescript
+// Direct support for graphql-ws added in Apollo Client Web v3.5.10.
+// See the next section for older versions.
+import { GraphQLWsLink } from "@apollo/client/link/subscriptions";
+import { createClient } from 'graphql-ws';
+
+const link = new GraphQLWsLink(createClient({
+  url: 'ws://where.is:4000/graphql',
+  connectionParams: () => {
+    const session = getSession();
+    if (!session) {
+      return {};
+    }
+    return {
+      Authorization: `Bearer ${session.token}`,
+    };
+  },
+});
+```
+
+</details>
+
+<details id="apollo-client">
+<summary><a href="#apollo-client">🔗</a> Client usage with <a href="https://www.apollographql.com/docs/react/">Apollo Client Web</a> prior to v3.5.10</summary>
```

Comment:
This could be joined into a single accordion. When expanded, show the 2 different approaches (recommended one on top).

Reply:
Makes sense. I combined them and I also changed the API of the old version to match the built-in version (named it `GraphQLWsLink`, takes a `Client` as an argument instead of `ClientOptions`) so that the usage code could be shared.